.gitignore

```diff
@@ -2,5 +2,7 @@
 literallycanvas/
 *.sublime-workspace
 node_modules/
+core/
+core/fontmetrics.js
```

Comment:
Unfortunately the main `fontmetrics` package has an XSS bug, so this should be reverted. I don't trust it to continue to be correct.

Reply:
Oh I see.... but I have the same version you had in the core, so it should be pulling in [this one](https://github.com/literallycanvas/literallycanvas-core/blob/master/src/fontmetrics.js) when you run buildcore.

Reply:
Oh, ok! Then can't we just remove it from the repo?

Reply:
I mean rather than adding it to gitignore, since it's gone anyway.

Reply:
I'll try again, for some reason something kept tracking it's changes, despite the fact that the whole core folder is supposed to be ignored. I'll check again.

Reply:
Ok, seems fine now.... just some git strangeness.